src/test/java/selenium/AlertsTest.java

```diff
@@ -0,0 +1,77 @@
+package selenium;
+
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import testbase.TestBase;
+
+import static constants.ByConstants.*;
+import static constants.TestConstants.*;
+
+@Slf4j
+public class AlertsTest extends TestBase {
+    @Test
+    public void testJavaScriptAlertBox() {
+        log.info("Open the URL: {}", ALERTS_TEST_URL);
+        driver().get(ALERTS_TEST_URL);
+
+        log.info("Click the 'Click me!' button for the JavaScript Alert Box");
+        driver().findElement(ALERT_BOX_BUTTON).click();
+
+        log.info("Switch to the alert");
+        Alert alert = driver().switchTo().alert();
+
+        log.info("Get the alert text");
+        String alertText = alert.getText();
+
+        log.info("Verify the alert text is '{}'", ALERT_TEXT);
+        Assert.assertEquals(alertText, ALERT_TEXT);
```

Comment:
always use an assertion message for each assertions as an extra parameter, in this case it will be like "Alert text mismatch!"

Reply:
Done!